client/src/global/components/atomic/EntityThumbnail/index.js

```diff
@@ -16,20 +17,30 @@ export default function EntityThumbnail({
   parentView = false,
   isListItem = false
 }) {
+  let routeName = "frontendProjectDetail";
+
+  if (entity.type === "journals") {
+    routeName = "frontendJournalDetail";
+  } else if (entity.type === "projectCollections") {
+    routeName = "frontendProjectCollection";
+  }
```

Comment:
Since this is a global generic component, I wonder if we should just make the route name a prop.

Reply:
I understand your concern, but this component isn't truly generic. It is generic to entities, which have a type and a specific route name. (The component originally went to "frontendProjectDetail", no matter what entity you gave it.) Since this component gets the full entity object with type, it seemed unnecessary to made a developer pass in the route name. 

I could see passing in the route name if there's a chance we want the preview to go somewhere other than the entity type's detail page.